README.md

```diff
@@ -16,6 +16,8 @@ You can import the generated bundle to use the whole library generated by this s
 ```javascript
 import * as tillhub from '@tillhub/javascript-sdk'
 
+var th = new tillhub.TillhubClient();
```

Comment:
Good point, but think we need to change another line there.
It should be

```
import th from '@tillhub/javascript-sdk'
```
instead of
```
import * as tillhub from '@tillhub/javascript-sdk'
```